internal/policies/fetcher.go

```diff
@@ -134,8 +152,8 @@ func (f *Fetcher) getAdmissionPolicies(namespace string) ([]policiesv1.Admission
 func newClient() (client.Client, error) {
 	config := ctrl.GetConfigOrDie()
 	customScheme := scheme.Scheme
-	customScheme.AddKnownTypes(schema.GroupVersion{Group: kubewardenPoliciesGroup, Version: kubewardenPoliciesVersion}, &policiesv1.ClusterAdmissionPolicy{}, &policiesv1.AdmissionPolicy{}, &policiesv1.ClusterAdmissionPolicyList{}, &policiesv1.AdmissionPolicyList{})
-	metav1.AddToGroupVersion(customScheme, schema.GroupVersion{Group: kubewardenPoliciesGroup, Version: kubewardenPoliciesVersion})
+	customScheme.AddKnownTypes(schema.GroupVersion{Group: constants.KubewardenPoliciesGroup, Version: constants.KubewardenPoliciesVersion}, &policiesv1.ClusterAdmissionPolicy{}, &policiesv1.AdmissionPolicy{}, &policiesv1.ClusterAdmissionPolicyList{}, &policiesv1.AdmissionPolicyList{})
```

Comment:
nitpicking, can we indent a bit the code to make it more readable? like you have done inside of `report/client.go`

Reply:
done in https://github.com/kubewarden/audit-scanner/pull/40/commits/2db512b545e1262c1288a96bc55252ee2ac676ca